### PR TITLE
New Cal CSVs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New cal csvs for refurbed OPTAAs, CTDBPs & PRESFs, plus cal csvs for 6 in-house 2pt DOSTA cals
Note - most of the CTDBP & PRESF QCTs have not been reviewed yet - I added the source_file for each in the format it will be once released to Alfresco (our standard format), but for review refer to the associated QCT .txt or .cap file in Vault.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/CTDBPC/CGINS-CTDBPC-06841__20210603.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+16P64183-6841,CC_i,-2.389312e-04,
+16P64183-6841,CC_j,3.467975e-05,
+16P64183-6841,CC_pa0,3.445200e-02,
+16P64183-6841,CC_pa1,4.870368e-04,
+16P64183-6841,CC_pa2,-2.816965e-12,
+16P64183-6841,CC_ptca0,5.251895e+05,
+16P64183-6841,CC_ptca1,3.898020e+00,
+16P64183-6841,CC_ptca2,-3.731887e-02,
+16P64183-6841,CC_ptcb0,2.503125e+01,
+16P64183-6841,CC_ptcb1,5.000000e-05,
```
Comment (IDCalTech) [inline calibration/CTDBPC/CGINS-CTDBPC-06841__20210603.csv:19]: Looks good

File: calibration/CTDBPC/CGINS-CTDBPC-50003__20210617.csv [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+16-50003,CC_pa2,-3.585731e-12,
+16-50003,CC_ptca0,5.245473e+05,
+16-50003,CC_ptca1,-3.008939e-01,
+16-50003,CC_ptca2,-1.045975e-01,
+16-50003,CC_ptcb0,2.502750e+01,
+16-50003,CC_ptcb1,-9.000000e-04,
+16-50003,CC_ptcb2,0.000000e+00,
+16-50003,CC_ptempa0,-6.002286e+01,
+16-50003,CC_ptempa1,5.486440e+01,
+16-50003,CC_ptempa2,-6.438470e-01,
```
Comment (IDCalTech) [inline calibration/CTDBPC/CGINS-CTDBPC-50003__20210617.csv:23]: Looks good

File: calibration/CTDBPD/CGINS-CTDBPD-07209__20210603.csv [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+16P71174-7209,CC_pa2,9.052235e-12,
+16P71174-7209,CC_ptca0,5.248600e+05,
+16P71174-7209,CC_ptca1,6.469763e+00,
+16P71174-7209,CC_ptca2,-1.151138e-01,
+16P71174-7209,CC_ptcb0,2.510600e+01,
+16P71174-7209,CC_ptcb1,-9.999999e-04,
+16P71174-7209,CC_ptcb2,0.000000e+00,
+16P71174-7209,CC_ptempa0,-6.098543e+01,
+16P71174-7209,CC_ptempa1,5.314285e+01,
+16P71174-7209,CC_ptempa2,-3.689311e-01,
```
Comment (IDCalTech) [inline calibration/CTDBPD/CGINS-CTDBPD-07209__20210603.csv:23]: Looks good

File: calibration/CTDBPD/CGINS-CTDBPD-50110__20210603.csv [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+16-50110,CC_pa2,8.755432e-11,
+16-50110,CC_ptca0,5.210456e+05,
+16-50110,CC_ptca1,-7.146705e+01,
+16-50110,CC_ptca2,1.600145e-01,
+16-50110,CC_ptcb0,2.480888e+01,
+16-50110,CC_ptcb1,-2.250000e-04,
+16-50110,CC_ptcb2,0.000000e+00,
+16-50110,CC_ptempa0,1.784527e+02,
+16-50110,CC_ptempa1,-7.736322e+01,
+16-50110,CC_ptempa2,4.149924e-01,
```
Comment (IDCalTech) [inline calibration/CTDBPD/CGINS-CTDBPD-50110__20210603.csv:23]: Looks good

File: calibration/CTDBPE/CGINS-CTDBPE-50004__20210530.csv [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+16-50004,CC_pa2,1.575161e-11,
+16-50004,CC_ptca0,5.243915e+05,
+16-50004,CC_ptca1,5.714449e+01,
+16-50004,CC_ptca2,-8.997587e-01,
+16-50004,CC_ptcb0,2.502037e+01,
+16-50004,CC_ptcb1,-3.250000e-04,
+16-50004,CC_ptcb2,0.000000e+00,
+16-50004,CC_ptempa0,-7.473688e+01,
+16-50004,CC_ptempa1,5.084569e+01,
+16-50004,CC_ptempa2,-5.503493e-01,
```
Comment (IDCalTech) [inline calibration/CTDBPE/CGINS-CTDBPE-50004__20210530.csv:23]: Looks good.

File: calibration/DOSTAD/CGINS-DOSTAD-00126__20210830.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+126,CC_conc_coef,"[-2.973900E+00, 1.051900E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_126_Calibration_Certificate_2021-08-30.pdf"
+126,CC_csv,"[2.68474E-03, 1.17084E-04, 2.23543e-06, 1.98627E+02, -2.16587E-01, -4.89511E+01, 3.94502E+00]",
```
Comment (IDCalTech) [inline calibration/DOSTAD/CGINS-DOSTAD-00126__20210830.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00225__20210831.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+225,CC_conc_coef,"[-0.99599E+00, 1.0536E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_225_Calibration_Certificate_2021-08-31.pdf"
+225,CC_csv,"[2.91240E-03, 1.23573E-04, 2.33717E-06, 1.50442E+02, -2.04692E-01, -3.61233E+01, 2.91530E+00]",
```
Comment (IDCalTech) [inline calibration/DOSTAD/CGINS-DOSTAD-00225__20210831.csv:3]: Looks good.

File: calibration/DOSTAD/CGINS-DOSTAD-00385__20210907.csv [hunk 8]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+385,CC_conc_coef,"[-9.0194E-01, 1.0588E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_385_Calibration_Certificate_2021-09-07.pdf"
+385,CC_csv,"[2.74350E-03, 1.15555E-04, 2.23354E-06, 2.30688E+02, -3.72162E-01, -5.63315E+01, 4.55591E+00]",
```
Comment (IDCalTech) [inline calibration/DOSTAD/CGINS-DOSTAD-00385__20210907.csv:3]: Looks good.

File: calibration/DOSTAD/CGINS-DOSTAD-00410__20210831.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+410,CC_conc_coef,"[-2.3407E+00, 1.09E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_410_Calibration_Certificate_2021-08-31.pdf"
+410,CC_csv,"[2.54703E-03, 1.08188E-04, 2.14570E-06, 2.29342E+02, -3.54081E-01, -5.59270E+01, 4.56498E+00]",
```
Comment (IDCalTech) [inline calibration/DOSTAD/CGINS-DOSTAD-00410__20210831.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00502__20210902.csv [hunk 10]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+502,CC_conc_coef,"[9.3247E-01, 9.9072E-01]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_502_Calibration_Certificate_2021-09-02.pdf"
+502,CC_csv,"[2.65279E-03, 1.13001E-04, 2.24904E-06, 9.12452E+01, -1.45210E-01, -2.24209E+01, 1.81645E+00]",
```
Comment (IDCalTech) [inline calibration/DOSTAD/CGINS-DOSTAD-00502__20210902.csv:3]: Looks good.

File: calibration/DOSTAD/CGINS-DOSTAD-00663__20210827.csv [hunk 11]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+663,CC_conc_coef,"[-0.272430E+00, 1.078700E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_663_Calibration_Certificate_2021-08-27.pdf"
+663,CC_csv,"[2.81476E-03, 1.18957E-04, 2.37181E-06, 2.30050E+02, -3.88265E-01, -5.77627E+01, 4.55125E+00]",
```
Comment (IDCalTech) [inline calibration/DOSTAD/CGINS-DOSTAD-00663__20210827.csv:3]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00206__20210813.csv [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+ACS-206,CC_tbins,"[1.274165, 2.455263, 3.481923, 4.440556, 5.549048, 6.516667, 7.504167, 8.491600, 9.499600, 10.484167, 11.483600, 12.487600, 13.504000, 14.501667, 15.490385, 16.516400, 17.508800, 18.486667, 19.488400, 20.495600, 21.499600, 22.486000, 23.501111, 24.513750, 25.501200, 26.493462, 27.497200, 28.482308, 29.519583, 30.508000, 31.486538, 32.510833, 33.501071, 34.457500]",
+ACS-206,CC_taarray,SheetRef:CC_taarray,
+ACS-206,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00206__20210813.csv:9]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00206__20210813__CC_taarray.ext [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+-0.000399,0.001587,0.001786,0.001968,0.002097,0.002234,0.002305,0.002299,0.002404,0.002571,0.002597,0.002656,0.002697,0.002749,0.002763,0.002878,0.002682,0.002353,0.002214,0.001849,0.001585,0.001123,0.000716,0.000399,0,-0.000449,-0.00081,-0.001133,-0.001532,-0.001731,-0.002099,-0.002246,-0.002427,-0.002688
+-0.000759,0.001173,0.0014,0.001612,0.001678,0.001787,0.001874,0.001929,0.002026,0.002132,0.002292,0.002402,0.002416,0.002408,0.002495,0.002618,0.002386,0.00212,0.002018,0.001701,0.001419,0.001112,0.00068,0.000385,0,-0.000437,-0.000778,-0.001086,-0.001393,-0.001611,-0.0019,-0.002194,-0.002277,-0.002531
+-0.001206,0.000744,0.000954,0.001097,0.001271,0.001464,0.001549,0.001522,0.001624,0.001847,0.001942,0.002089,0.002066,0.002104,0.00224,0.002401,0.002158,0.001906,0.001844,0.00146,0.001326,0.000994,0.000619,0.000309,0,-0.000362,-0.000603,-0.000954,-0.001328,-0.001541,-0.001799,-0.001969,-0.002181,-0.002396
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00206__20210813__CC_taarray.ext:86]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00206__20210813__CC_tcarray.ext [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+0.002891,0.002669,0.002442,0.00229,0.002233,0.001881,0.001905,0.001797,0.001477,0.001604,0.001346,0.00115,0.001099,0.001199,0.000998,0.000829,0.000708,0.000581,0.000694,0.000579,0.000505,0.000423,0.000208,0.000108,0,3.1e-05,-0.00018,-0.000554,-0.000942,-0.001314,-0.001487,-0.001729,-0.002067,-0.002505
+0.003234,0.003509,0.003274,0.003015,0.003101,0.002873,0.002659,0.002537,0.002244,0.001918,0.001648,0.001718,0.001419,0.001352,0.001275,0.000958,0.000896,0.00076,0.000662,0.00055,0.00046,0.000325,0.000142,0.000101,0,-2.9e-05,-0.000286,-0.000489,-0.001035,-0.00104,-0.001442,-0.001692,-0.002124,-0.002517
+0.002759,0.002745,0.002488,0.002431,0.002412,0.002023,0.001817,0.001835,0.001817,0.00166,0.001609,0.001593,0.001508,0.001397,0.001227,0.001037,0.000805,0.000658,0.000602,0.000667,0.000412,0.000345,0.000113,4.3e-05,0,-9.6e-05,-0.000344,-0.000564,-0.000813,-0.001146,-0.001457,-0.001618,-0.002076,-0.002627
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00206__20210813__CC_tcarray.ext:86]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00123__20210812.csv [hunk 15]
```diff
@@ -0,0 +1,9 @@
...
+ACS-123,CC_tbins,"[0.803385, 1.274369, 2.465357, 3.420909, 4.560556, 5.463600, 6.506538, 7.472727, 8.521667, 9.484583, 10.490000, 11.485652, 12.498750, 13.490000, 14.481600, 15.512609, 16.485000, 17.495833, 18.512174, 19.522963, 20.453200, 21.536087, 22.446071, 23.491429, 24.528000, 25.479500, 26.542963, 27.427308, 28.528571, 29.507879, 30.489000, 31.542414, 32.457083, 33.565000, 34.395217]",
+ACS-123,CC_taarray,SheetRef:CC_taarray,
+ACS-123,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00123__20210812.csv:9]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00123__20210812__CC_taarray.ext [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+-0.010764,-0.008179,-0.007617,-0.007486,-0.007313,-0.007004,-0.006766,-0.006532,-0.006351,-0.006154,-0.005816,-0.005488,-0.005289,-0.004932,-0.004471,-0.004097,-0.00385,-0.0034,-0.003095,-0.002511,-0.002103,-0.001583,-0.001217,-0.000894,-0.000468,0,0.000215,0.000637,0.000797,0.00115,0.001397,0.001262,0.001394,0.001566,0.001652
+-0.010628,-0.008122,-0.007547,-0.007533,-0.007273,-0.006846,-0.006707,-0.006397,-0.006265,-0.005953,-0.005636,-0.005443,-0.005138,-0.00493,-0.004407,-0.003979,-0.00368,-0.003214,-0.003014,-0.00238,-0.002064,-0.001533,-0.001144,-0.000773,-0.000243,0,0.000322,0.000843,0.00101,0.001246,0.001584,0.001472,0.001666,0.001766,0.001782
+-0.010919,-0.008459,-0.007894,-0.007757,-0.00758,-0.007182,-0.007004,-0.006778,-0.006577,-0.006332,-0.005928,-0.005686,-0.005378,-0.005056,-0.004576,-0.004196,-0.003885,-0.003449,-0.003227,-0.002577,-0.002183,-0.001643,-0.001243,-0.00096,-0.000467,0,0.000265,0.000712,0.000846,0.001111,0.001514,0.001408,0.001538,0.001591,0.001608
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00123__20210812__CC_taarray.ext:83]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00123__20210812__CC_tcarray.ext [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+0.009612,0.009233,0.008708,0.008488,0.008216,0.007805,0.007422,0.007206,0.00683,0.006424,0.006097,0.005358,0.004802,0.004211,0.003744,0.003438,0.002842,0.002304,0.001833,0.001323,0.000827,0.000404,0.000146,0.000308,-2.6e-05,0,-3.3e-05,-0.000216,-0.000251,-7.9e-05,-2.1e-05,-0.000187,-8e-06,-9.8e-05,-0.000214
+0.009801,0.009849,0.009432,0.009078,0.008949,0.008656,0.008175,0.007848,0.007347,0.006603,0.006211,0.00586,0.005374,0.004957,0.004188,0.003371,0.002575,0.002168,0.001705,0.001289,0.000916,0.000435,0.000328,7.2e-05,-0.000188,0,-0.000227,-0.000249,-0.000289,-0.000267,-0.000371,-0.000483,-0.000393,-0.000341,-0.000423
+0.010254,0.009845,0.009436,0.009328,0.008858,0.00869,0.008208,0.007912,0.007551,0.007178,0.006736,0.006221,0.005668,0.005026,0.004479,0.004023,0.003193,0.002665,0.002217,0.001593,0.00097,0.000638,0.000247,0.000241,-3.8e-05,0,-0.000138,3.4e-05,-0.000286,-0.000318,-0.000392,-0.000388,-0.000311,-0.00031,-0.000251
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00123__20210812__CC_tcarray.ext:83]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00222__20210817.csv [hunk 18]
```diff
@@ -0,0 +1,9 @@
...
+ACS-222,CC_tbins,"[0.702438, 1.323836, 2.463448, 3.444783, 4.531875, 5.468333, 6.504400, 7.514400, 8.498333, 9.502083, 10.494167, 11.487917, 12.488800, 13.481250, 14.474000, 15.500000, 16.505600, 17.500800, 18.487917, 19.506000, 20.521200, 21.500833, 22.505200, 23.482800, 24.484545, 25.561429, 26.477586, 27.498000, 28.468000, 29.500400, 30.515200, 31.500357, 32.470000, 33.532500, 34.467917]",
+ACS-222,CC_taarray,SheetRef:CC_taarray,
+ACS-222,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00222__20210817.csv:9]: Looks good

File: calibration/PRESFB/CGINS-PRESFB-01329__20210620.csv [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+26P64515-1329,CC_pressure_offset_calibration_coefficient,-3.250000e-01,
+26P64515-1329,CC_slope_correction_factor,1,Constant
+26P64515-1329,CC_t1,2.777282e+01,
+26P64515-1329,CC_t2,3.911380e-01,
+26P64515-1329,CC_t3,1.752851e+01,
+26P64515-1329,CC_t4,3.109619e+01,
+26P64515-1329,CC_u0,5.827424e+00,
+26P64515-1329,CC_y1,-3.845795e+03,
+26P64515-1329,CC_y2,-1.082941e+04,
+26P64515-1329,CC_y3,0.000000e+00,
```
Comment (IDCalTech) [inline calibration/PRESFB/CGINS-PRESFB-01329__20210620.csv:19]: Looks good

File: calibration/PRESFC/CGINS-PRESFC-01388__20210620.csv [hunk 20]
```diff
@@ -0,0 +1,19 @@
...
+26P76848-1388,CC_pressure_offset_calibration_coefficient,-8.248000e-01,
+26P76848-1388,CC_slope_correction_factor,1,
+26P76848-1388,CC_t1,3.036646e+01,
+26P76848-1388,CC_t2,7.435300e-01,
+26P76848-1388,CC_t3,5.042604e+01,
+26P76848-1388,CC_t4,1.440226e+02,
+26P76848-1388,CC_u0,5.858992e+00,
+26P76848-1388,CC_y1,-3.948355e+03,
+26P76848-1388,CC_y2,-1.019385e+04,
+26P76848-1388,CC_y3,0.000000e+00,
```
Comment (IDCalTech) [inline calibration/PRESFC/CGINS-PRESFC-01388__20210620.csv:19]: Looks good

File: calibration/PRESFC/CGINS-PRESFC-01400__20210615.csv [hunk 21]
```diff
@@ -0,0 +1,19 @@
...
+26-1400,CC_pressure_offset_calibration_coefficient,-2.639000e-01,
+26-1400,CC_slope_correction_factor,1,Constant
+26-1400,CC_t1,3.035705e+01,
+26-1400,CC_t2,1.204163e+00,
+26-1400,CC_t3,4.801965e+01,
+26-1400,CC_t4,1.125149e+02,
+26-1400,CC_u0,5.772240e+00,
+26-1400,CC_y1,-3.999241e+03,
+26-1400,CC_y2,-1.075324e+04,
+26-1400,CC_y3,0.000000e+00,
```
Comment (IDCalTech) [inline calibration/PRESFC/CGINS-PRESFC-01400__20210615.csv:19]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00222__20210817__CC_taarray.ext [hunk 22]
```diff
@@ -0,0 +1,90 @@
...
+0.004189,0.002963,0.002619,0.002419,0.002388,0.002393,0.002127,0.001989,0.001983,0.001699,0.001614,0.001271,0.001327,0.001132,0.000829,0.000936,0.000922,0.000851,0.000605,0.000441,0.000338,0.000395,0.000233,0.000112,4.5e-05,0,-0.000182,-0.00035,-0.000319,-0.000535,-0.000356,-0.000379,-0.000423,-0.00058,-0.000548
+0.003726,0.002632,0.002231,0.00212,0.002076,0.002046,0.001855,0.001725,0.001753,0.001462,0.001366,0.001125,0.001074,0.000974,0.000684,0.000793,0.000758,0.000716,0.000501,0.000354,0.000393,0.000266,0.000203,0.000114,7.9e-05,0,-0.000184,-0.000406,-0.000322,-0.000608,-0.00044,-0.000438,-0.000429,-0.000558,-0.000588
+0.003514,0.002355,0.002061,0.001981,0.00183,0.001896,0.001693,0.001646,0.001557,0.001316,0.001307,0.000999,0.001064,0.00097,0.000659,0.00076,0.000809,0.000698,0.000477,0.000345,0.000411,0.000314,0.000164,0.000104,-3.4e-05,0,-0.000158,-0.000259,-0.000275,-0.000557,-0.000333,-0.000345,-0.000391,-0.000528,-0.000503
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00222__20210817__CC_taarray.ext:90]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00222__20210817__CC_tcarray.ext [hunk 23]
```diff
@@ -0,0 +1,90 @@
...
+-0.015714,-0.012355,-0.011073,-0.010276,-0.009635,-0.009207,-0.008617,-0.007918,-0.007233,-0.006703,-0.00596,-0.005299,-0.004254,-0.003912,-0.002716,-0.002528,-0.001905,-0.001589,-0.001392,-0.001053,-0.000806,-0.000515,-0.000322,-0.000254,9.1e-05,0,0.000141,0.000129,0.000314,0.000211,-9.5e-05,0.000135,3e-05,0.000184,-6.7e-05
+-0.015922,-0.012474,-0.011422,-0.010751,-0.010477,-0.010043,-0.009399,-0.008357,-0.007633,-0.006812,-0.005931,-0.005301,-0.004501,-0.003678,-0.003197,-0.002621,-0.002092,-0.001853,-0.001506,-0.001058,-0.000773,-0.000555,-0.000338,-9.3e-05,-8.4e-05,0,6.9e-05,0.000283,5.5e-05,-4e-05,0.000163,0.00011,0.000148,3e-05,-3.3e-05
+-0.016496,-0.014,-0.01217,-0.011506,-0.010347,-0.009914,-0.008699,-0.008358,-0.007795,-0.007111,-0.005765,-0.005197,-0.004856,-0.003881,-0.003127,-0.002882,-0.002166,-0.00201,-0.001576,-0.001057,-0.000835,-0.000801,-0.000418,-0.000554,-0.000521,0,-0.000229,-0.000145,-0.000246,-0.00049,-0.000573,-0.000417,-0.000613,-0.000592,-0.000973
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00222__20210817__CC_tcarray.ext:90]: Looks good

**Review by IDCalTech — APPROVED** [review]
Looks good

File: calibration/PRESFB/CGINS-PRESFB-01329__20210620.csv (hunk 19 above)
Comment (reedan88) [inline calibration/PRESFB/CGINS-PRESFB-01329__20210620.csv:19]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01352__20210614.csv [hunk 24]
```diff
@@ -0,0 +1,19 @@
...
+26P71826-1352,CC_pressure_offset_calibration_coefficient,-3.578000e-01,
+26P71826-1352,CC_slope_correction_factor,1,Constant
+26P71826-1352,CC_t1,2.787143e+01,
+26P71826-1352,CC_t2,4.900260e-01,
+26P71826-1352,CC_t3,1.718162e+01,
+26P71826-1352,CC_t4,1.343469e+01,
+26P71826-1352,CC_u0,5.825881e+00,
+26P71826-1352,CC_y1,-3.854568e+03,
+26P71826-1352,CC_y2,-1.065269e+04,
+26P71826-1352,CC_y3,0.000000e+00,
```
Comment (reedan88) [inline calibration/PRESFB/CGINS-PRESFB-01352__20210614.csv:19]: Looks good

File: calibration/PRESFB/CGINS-PRESFB-01387__20210616.csv [hunk 25]
```diff
@@ -0,0 +1,19 @@
...
+26P76848-1387,CC_pressure_offset_calibration_coefficient,-3.094000e-01,
+26P76848-1387,CC_slope_correction_factor,1,Constant
+26P76848-1387,CC_t1,2.767723e+01,
+26P76848-1387,CC_t2,3.907120e-01,
+26P76848-1387,CC_t3,1.805891e+01,
+26P76848-1387,CC_t4,2.615364e+01,
+26P76848-1387,CC_u0,5.828209e+00,
+26P76848-1387,CC_y1,-3.920251e+03,
+26P76848-1387,CC_y2,-1.047477e+04,
+26P76848-1387,CC_y3,0.000000e+00,
```
Comment (reedan88) [inline calibration/PRESFB/CGINS-PRESFB-01387__20210616.csv:19]: Looks good.

File: calibration/PRESFC/CGINS-PRESFC-01388__20210620.csv (hunk 20 above)
Comment (reedan88) [inline calibration/PRESFC/CGINS-PRESFC-01388__20210620.csv:19]: Looks good

**Review by reedan88 — CHANGES_REQUESTED** [review]
I have a few questions about an OPTAA significant digits and a couple of CTDBP serial numbers.

File: calibration/CTDBPC/CGINS-CTDBPC-06841__20210603.csv [hunk 26]
```diff
@@ -0,0 +1,23 @@
...
+16P64183-6841,CC_ptcb2,0.000000e+00,
+16P64183-6841,CC_ptempa0,-6.449301e+01,
+16P64183-6841,CC_ptempa1,5.165168e+01,
+16P64183-6841,CC_ptempa2,-2.897289e-01,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply calibration/CTDBPC/CGINS-CTDBPC-06841__20210603.csv:23]: SN is correct - there are just some weird ones.  And we determined that using the last 4 digits with a leading zero was the way to go for the filenames.

File: calibration/DOSTAD/CGINS-DOSTAD-00126__20210830.csv (hunk 6 above)
Comment (rtravis-whoi) [reply calibration/DOSTAD/CGINS-DOSTAD-00126__20210830.csv:3]: I have been told from the start of my involvement in this that the little vs big E does not make a difference - there's a lot of variation throughout cal csvs and nothing's broken (as far as I know).

**Pull request merged**